packages/ember-glimmer/lib/environment.js

```diff
@@ -133,6 +137,18 @@ export default class Environment extends GlimmerEnvironment {
           wrapClassAttribute(args);
           return new CurlyComponentSyntax({ args, definition, templates });
         }
+      } else {
+        // Check if it's a keyword
+        let mappedKey = keywords[key];
+        if (mappedKey) {
+          if (mappedKey !== key) {
+            path = path.map((segment) => segment === key ? mappedKey : segment);
```

Comment:
I took a look at input, and to get input to work, we'll need to add the ability to choose the component based on the keyword, since input can be a checkbox or a textfield. I think we should just merge what I have now and we can add the dynamic component lookup when we get to input.